### ISSUE TITLE
Batch process for updating feeds

### ISSUE THREAD

**Issue by emmambd** [MobilityData/mobility-feed-api#22]
## **User Story**

As an API consumer, I need to get updated feeds, so that I know which producer URL is the most up-to-date and my datasets are as recent as possible. 

## **Acceptance Criteria**

- Batch process should include all the steps for updating feeds outlined in [this flowchart](https://miro.com/app/board/uXjVM86LB1s=/?moveToWidget=3458764558860655921&cot=14)
![Public  Mobility Database API_ User Story Mapping - Flowchart for updating feeds](https://github.com/MobilityData/mobility-feed-api/assets/90632321/fe03e23a-f449-4d9c-a145-f1f0982f4d11)

## **Materials**

- [Current cronjob for storing latest datasets in the Mobility Database catalogs](https://github.com/MobilityData/mobility-database-catalogs/blob/main/.github/workflows/store_latest_datasets_cronjob.yml)

**Comment by davidgamez** [comment]
Depends on #16

**Comment by emmambd** [comment]
Duplicate of #16
